ArduPlane/RC_Channel.cpp

```diff
@@ -123,6 +123,20 @@ void RC_Channel_Plane::do_aux_function(const aux_func_t ch_option, const aux_swi
         do_aux_function_change_mode(Mode::Number::TAKEOFF, ch_flag);
         break;
 
+    case AUX_FUNC::AVOID_ADSB:
```

Comment:
There's enough code here to warrant using the adb singleton and putting this into `libraries/RC_Channel/RC_Channel.cpp`.  We'll probably want it there when Rover gets AIS support, too.